### ISSUE TITLE
теги в css

### ISSUE THREAD

**Issue by DmitriiMelnikOnix** [EradekoraIvan/REACT-onix.academy#5]
Лучше не использовать теги в css для стилизации 
```
.footer_group h3 {
    font-size: 16px;
    font-weight: 800;
    color: #FFFFFF;
} ...
```

Лучше используй классы. Это связано с тем как применяются стили. Есть байка, которую я никогда не проверял, что сначала браузер находит все `h3` а только потом фильтрует по `.footer_group` и это влияет на производительность
Но это не так важно как наглядность. Всегда лучше чтобы было явно видно как стилизируется тег. Так что лучше используй классы

**Comment by DmitriiMelnikOnix** [comment]
Но это скорее пожелание) И может быть не везде это оправдано

**Comment by EradekoraIvan** [comment]
Да, я заметил, что если обращаться через теги, то страдает модульность стилей и они применяются к элементам в других компонентах... но не всегда. В общем можно напортачить с весом селекторов.
